pkg/controller/utils/conditions.go

```diff
@@ -50,6 +50,7 @@ func SetAPISchemeCondition(
 					Reason:             reason,
 					Message:            message,
 					LastTransitionTime: now,
+					LastProbeTime:      now,
```

Comment:
This was error-ing in the logs. `LastProbeTime` needs a  value set

Reply:
this sounds like it will fix https://issues.redhat.com/browse/OSD-6789. I'll test this today.